array-manipulation-2/src/includes.js

```diff
@@ -0,0 +1,10 @@
+/* exported includes */
+
+function includes(array, value) {
+  for (var i = 0; i < array.length; i++) {
+    if (array.indexOf(value) === i) {
```

Comment:
You don't need to use `indexOf` here, you can just do this
`if (array[i] === value)`